MultisigIntegrationTests/TokenRegistryIntegrationTests.swift

```diff
@@ -18,40 +18,44 @@ class TokenRegistryIntegrationTests: XCTestCase {
     let notAToken: Address = "0x9bcd3162694994f67d49a8ead6e5a196c9dd2fd2"
 
     override func setUpWithError() throws {
-        XTAssertEqual(App.configuration.app.network, .rinkeby,
+        XCTAssertEqual(App.configuration.app.network, .rinkeby,
                       "The test users rinkeby addresses")
     }
 
     func testLoad() {
-        XCTAssertNil(registry[cryptoKitties])
-
         let exp = expectation(description: "Wait")
         registry.load {
             exp.fulfill()
         }
+        waitForExpectations(timeout: 3, handler: nil)
+
+        let sema = DispatchSemaphore(value: 0)
+        DispatchQueue.global().async {
 
-        XCTAssertNotNil(registry[cryptoKitties])
+            XCTAssertNotNil(self.registry["0x2FDFfE9Cad323d3922195292E8806987B1fc1AAD"])
+            sema.signal()
+        }
+        sema.wait()
     }
 
     func testFetch() {
         var tokens = [Token?]()
 
         let exp = expectation(description: "Wait")
         DispatchQueue.global().async { [unowned self] in
-            // hardcoded
-            tokens.append(self.registry[AddressRegistry.ether])
-            // pull from backend (AQER)
-            tokens.append(self.registry[cryptoKitties])
-            // same token, cached
-            tokens.append(self.registry[cryptoKitties])
-            // pull from blockchain (LINK)
-            tokens.append(self.registry[chainLinkNotInBackend])
-            // not a token -- should be recognized as ERC20 (this is a fallback logic)
-            tokens.append(self.registry[notAToken])
+            tokens.append(self.registry[.ether])
```

Comment:
As you explicitly capture self, do we still need to use self. ?

Reply:
yes, it's still not swift 5.2 :)